src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -217,6 +219,46 @@ export default {
 				this.open = false
 			}
 		},
+		afterShow() {
+			const picker = this.$refs.picker
+			const input = picker.$refs.search.$el.querySelector('input')
+
+			// add focus trap in modal
+			picker.$el.addEventListener('keydown', this.checkKeyEvent)
+
+			if (input) {
+				input.focus()
+			}
+
+		},
+		checkKeyEvent(event) {
+			const picker = this.$refs.picker
+
+			const focusableList = picker.$el.querySelectorAll(
+				'button, input'
+			)
+			// escape early if only 1 or no elements to focus
+			if (focusableList.length < 2 && event.key === 'Tab') {
+				event.preventDefault()
+				return
+			}
+			const last = focusableList.length - 1
+			if (
+				event.key === 'Tab'
+				&& event.shiftKey === false
+				&& event.target === focusableList[last]
+			) {
+				event.preventDefault()
```

Comment:
Comment so it can't be misunderstood.

`// Jump to first item when pressing tab on the latest item`

Reply:
✅